From 1481e8ff9f6b23eba2828d0def159a908c1ba53c Mon Sep 17 00:00:00 2001
From: Bian Jin chen <kenjc.bian@rock-chips.com>
Date: Fri, 12 Oct 2018 09:27:14 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3326-863-lp3-v10-avb: compatible with
 emmc and nand.

Android init will access "/proc/device-tree/firmware/android/boot_devices",
    then create symlinks for fstab:
dev/block/by-name/partition -> dev/block/type/${boot_devices}/by-name/partition

Change-Id: I1573686ea53a5226995b4d65a85636fcdd848dfa
Signed-off-by: Bian Jin chen <kenjc.bian@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10-avb.dts | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10-avb.dts b/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10-avb.dts
index ce9dc36787dd..174d05e37ccb 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10-avb.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10-avb.dts
@@ -13,6 +13,7 @@
 
 &firmware_android {
 	compatible = "android,firmware";
+	boot_devices = "ff390000.dwmmc,ff3b0000.nandc";
 	vbmeta {
 		compatible = "android,vbmeta";
 		parts = "vbmeta,boot,system,vendor,dtbo";
@@ -21,7 +22,7 @@
 		compatible = "android,fstab";
 		vendor {
 			compatible = "android,vendor";
-			dev = "/dev/block/platform/ff390000.dwmmc/by-name/vendor";
+			dev = "/dev/block/by-name/vendor";
 			type = "ext4";
 			mnt_flags = "ro,barrier=1,inode_readahead_blks=8";
 			fsmgr_flags = "wait,avb";
-- 
2.35.3

